app/lib/common/toolkit/html_editor/mentions/selected_mention_provider.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:acter/common/toolkit/html_editor/mentions/models/mention_type.dart';
```

Comment:
all this is a bit messy and I'd like to refactor this once we go over to search objects and rooms, too.